core/client/kt/verify.go

```diff
@@ -25,15 +25,16 @@ import (
 	"github.com/google/keytransparency/core/crypto/commitments"
 	"github.com/google/keytransparency/core/crypto/vrf"
 	"github.com/google/keytransparency/core/tree/sparse"
-	tv "github.com/google/keytransparency/core/tree/sparse/verifier"
```

Comment:
The grouping that we're using is:

- Standard packages.
- KT packages.
- Others.
- Proto.

I'm completely fine with a new scheme but we should apply it to all files.